### DISCUSSION TITLE
[Feature] Sending key combinations

### DISCUSSION THREAD

**Discussion by ghost** [86Box/86Box#3051] [Ideas]
I think there should be a special box where you can press/type (like `Ctrl+F12` or `Ctrl-F12`) any key combination you want.

  Plus, it would be nice to have special options to programmatically hold modifier keys inside emulated machine. For example, in a very quirky program we need to press **Ctrl+F12**. How can we do that?

Currently, we can only send **Left Alt** by pressing **Right Ctrl** if we enable this option.

**Comment by tongpeifu-yh** [comment]
Quite easy to change the shortcut. It is defined in Qt ui file 'qt_mainwindow.ui' so just open it with qtcreater and you can can see what to change.
